src/ontology/mondo-edit.obo

```diff
@@ -531789,21 +531769,35 @@ is_a: MONDO:0016120 ! myotonic syndrome
 [Term]
 id: MONDO:0056795
 name: X-linked spermatogenic failure 1
+subset: gard_rare {source="MONDO:GARD"}
 subset: gard_rare {source="GARD:8406", source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: obsoletion_candidate
```

Comment:
this "obsoletion_candidate" subset should be removed.

Reply:
done